pages/_document.tsx

```diff
@@ -216,18 +218,14 @@ class MyDocument extends Document {
           <script async type="text/javascript" src="/static/scripts/form-polyfills.js"></script>
           {GoogleTagScript}
         </CustomHead>
+        <noscript>
+          <style type="text/css">{noJsScss}</style>
+          <JSDisabled
+            formProps={this.props.__NEXT_DATA__.props.pageProps.formRecord}
+            lang={this.props.__NEXT_DATA__.locale}
```

Comment:
If the JS not enabled page will always display content if both languages should not we remove the `lang` prop?